.gitignore

```diff
@@ -1,20 +1,29 @@
-*#
+# Python
 *.py[co]
+.virtualenv
+_venv
+.pytest_cache/
+
+# Node
+node_modules
+
+# WPT repo stuff
+/MANIFEST.json
+
+testharness_runner.html
+!/tools/wptrunner/wptrunner/testharness_runner.html
+
+_certs
+config.json
+
+# Various OS/editor specific files
```

Comment:
Do we want to add a comment about some vague policy about adding stuff here? I mean, I'm assuming we don't just want to add everything from https://github.com/github/gitignore/tree/master/Global into here.

Reply:
"emacs is the one true editor" seems like a good policy to me :p

I expect it's not worth thinking too hard about this and we should just let the section grow in an ad-hoc way.